snap/hooks/install

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh -e
+
+cp -r $SNAP/tempest_test_lists $SNAP_DATA/tempest_test_lists
```

Comment:
AS we discussed the install hook is not enough if we do snap refresh it will not work and we need that to be able to update lists with new revisions.

Reply:
It's not enough to use dump plugin with organize to bake those files in snap itself?

Reply:
Yes, dump with organize also works. I updated it.